src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSScoreArgumentCollection.java

```diff
@@ -15,19 +15,30 @@
             fullName = "taxonomicDatabasePath")
     public String taxonomyDatabasePath;
 
-    @Argument(doc = "Alignment coverage score threshold, as a fraction of the read length (between 0 and 1)",
-            fullName = "scoreMinCoverage",
+    @Argument(doc = "Alignment identity score threshold, as a fraction of the read length (between 0 and 1).",
+            fullName = "scoreMinIdentity",
```

Comment:
I see the value of using constants for the names. I'll consider it as a future PR if I have some time because I would want to make this change throughout all of the other tools as well.